src/lib/provable/option.ts

```diff
@@ -104,6 +105,10 @@ function Option<A extends ProvableType>(
       return this.value;
     }
 
+    assertNone(): void {
```

Comment:
nit: we could pass in an optional `message?: string` argument to the function and pass it to `this.isSome.assertFalse(message)` so that a user can specify a optional error message on faliure (see `assertSome` above)